bash/04_export.sh

```diff
@@ -34,6 +34,9 @@ mkdir -p output
     CSV_export corrections_reference &
     CSV_export _manual_corrections manual_corrections &
 
+    display "Export A2 units for review by housing"
+    CSV_export UNITS_A2_devdb
```

Comment:
calling this table `UNITS_` breaks previous naming conventions (see above).  Maybe this is called `EXPORT_` or?  Naming is hard, I'm happy to discuss.

Reply:
That makes sense, I think the table name can be improved. 
My understanding is that this is more of a one-off for housing than something we will consistently export? An important question to answer to figure out a good name

Reply:
EXPECT_A2_devdb